bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -507,12 +507,17 @@ func (s *ksServer) getRegistryUri(registry *RegistryConfig) (string, error) {
 }
 
 func runCmd(rawcmd string) error {
-	cmd := exec.Command("sh", "-c", rawcmd)
-	err := cmd.Run()
-	if err != nil {
-		log.Errorf("Error occrued during execute cmd %v. Error: %v", rawcmd, err)
+	for retry := 0; ; retry++ {
+		cmd := exec.Command("sh", "-c", rawcmd)
+		result, err := cmd.CombinedOutput()
+		if err == nil || retry > 5 {
+			if err != nil {
+				return fmt.Errorf("Error occrued during execute cmd %v. Error: %v", rawcmd, string(result))
```

Comment:
this will only give you the last error. you might want to collect the error strings for each retry?